### DISCUSSION TITLE
Want to contribute

### DISCUSSION THREAD

**Discussion by loveneetsingh4** [pmndrs/react-spring#1778] [Support]
Hello all,
  This is loveneet senior software engineer from india want to contribute in your project

**Comment by joshuaellis** [comment]
Hey @internwt, my biggest advice would be check out the either an issue from the project board – https://github.com/pmndrs/react-spring/projects/2 or an issue from the `issues` section and get working! We have a channel in the pmndrs discord – https://discord.com/invite/ZZjjNvJ if you need any help on the issue, or feel free to @ me.
